[linux] npm ci fails on ubuntu 16.04.5, node.js v10.13.0

`npm ci` fails against commit 6aa8898b619a3ab888151d96f7489db03ae97a9c 

Dev environment is ubuntu 16.04.5, node.js v10.13.0

The error messages are:
```
> onnxjs@0.1.0 prepare /home/nhu/code/onnxjs
> tsc && node tools/build

test/test-runner.ts:144:47 - error TS2345: Argument of type 'NamedTensor[]' is not assignable to parameter of type 'Tensor[] | Map<string, Tensor>'.
  Type 'NamedTensor[]' is not assignable to type 'Tensor[]'.
    Type 'NamedTensor' is not assignable to type 'Tensor'.
      Property 'data' is missing in type 'NamedTensor'.

144     const outputs = await context.session.run(testCase.inputs!);
                                                  ~~~~~~~~~~~~~~~~

test/test-runner.ts:150:57 - error TS2339: Property 'type' does not exist on type 'NamedTensor'.

150       Logger.verbose('TestRunner', `   '${i.name}': ${i.type}[${i.dims.join(',')}]`);
                                                            ~~~~

test/test-runner.ts:150:67 - error TS2339: Property 'dims' does not exist on type 'NamedTensor'.

150       Logger.verbose('TestRunner', `   '${i.name}': ${i.type}[${i.dims.join(',')}]`);
                                                                      ~~~~

test/test-runner.ts:153:22 - error TS7006: Parameter 't' implicitly has an 'any' type.

153     outputs.forEach((t, name) => {
                         ~

test/test-runner.ts:153:25 - error TS7006: Parameter 'name' implicitly has an 'any' type.

153     outputs.forEach((t, name) => {
                            ~~~~

test/test-runner.ts:284:68 - error TS2345: Argument of type 'NamedTensor[]' is not assignable to parameter of type 'Tensor[]'.

284     this.checkTensorResult(expected.map(i => actual.get(i.name)!), expected);
                                                                       ~~~~~~~~

test/test-types.ts:6:22 - error TS2307: Cannot find module '../lib/Tensor'.

6 import {Tensor} from '../lib/Tensor';
                       ~~~~~~~~~~~~~~~

```

--- a/src/wasm-ops/conv.cpp
+++ b/src/wasm-ops/conv.cpp
@@ -4,8 +4,8 @@
 #include "conv.h"
 #include "common.h"
 #include "gemm.h"
-#include <Eigen\Core>
-#include <Eigen\Dense>
+#include <Eigen/Core>
+#include <Eigen/Dense>
 #include <vector>
 
 // Wasm interop method

--- a/src/wasm-ops/gemm.cpp
+++ b/src/wasm-ops/gemm.cpp
@@ -3,8 +3,8 @@
 
 #include "gemm.h"
 #include "common.h"
-#include <Eigen\Core>
-#include <Eigen\Dense>
+#include <Eigen/Core>
+#include <Eigen/Dense>
 
 // Wasm interop method
 void gemm_f32(void *data) {

--- a/test/test-types.ts
+++ b/test/test-types.ts
@@ -3,7 +3,7 @@
 
 import {Attribute} from '../lib/attribute';
 import {Logger} from '../lib/instrument';
-import {Tensor} from '../lib/Tensor';
+import {Tensor} from '../lib/tensor';
 
 export declare namespace Test {
   export interface NamedTensor extends Tensor {

--- a/tools/build.ts
+++ b/tools/build.ts
@@ -24,6 +24,7 @@ const DEPS = path.join(ROOT, 'deps');
 const DEPS_EIGEN = path.join(DEPS, 'eigen');
 const DEPS_EMSDK = path.join(DEPS, 'emsdk');
 const DEPS_EMSDK_EMSCRIPTEN = path.join(DEPS_EMSDK, 'emscripten');
+const EMSDK_BIN = path.join(DEPS_EMSDK, 'emsdk');
 const SRC = path.join(ROOT, 'src');
 const SRC_WASM_BUILD_CONFIG = path.join(SRC, 'wasm-build-config.json');
 const OUT = path.join(ROOT, 'dist');
@@ -91,7 +92,7 @@ if (!buildWasm) {
   logger.info('Build.Wasm', '(1/4) Setting up emsdk...');
   if (!fs.existsSync(DEPS_EMSDK_EMSCRIPTEN)) {
     logger.info('Build.Wasm', 'Installing emsdk...');
-    const install = spawnSync('emsdk install latest', {shell: true, stdio: 'inherit', cwd: DEPS_EMSDK});
+    const install = spawnSync(`${EMSDK_BIN} install latest`, {shell: true, stdio: 'inherit', cwd: DEPS_EMSDK});
     if (install.status !== 0) {
       if (install.error) {
         console.error(install.error);
@@ -101,7 +102,7 @@ if (!buildWasm) {
     logger.info('Build.Wasm', 'Installing emsdk... DONE');
 
     logger.info('Build.Wasm', 'Activating emsdk...');
-    const activate = spawnSync('emsdk activate latest', {shell: true, stdio: 'inherit', cwd: DEPS_EMSDK});
+    const activate = spawnSync(`${EMSDK_BIN} activate latest`, {shell: true, stdio: 'inherit', cwd: DEPS_EMSDK});
     if (activate.status !== 0) {
       if (activate.error) {
         console.error(activate.error);

